### ISSUE TITLE
对大视频（ts）文件进行Seek操作，Chrome浏览器下载文件过程中报【Caution: request is not finished yet】警告，导致Seek卡住

### ISSUE THREAD

**Issue by liminglu** [xqq/mpegts.js#118]
作者你好，

使用版本1.7.3
在播放视频文件（时长：75分钟），概率性从2s seek 15分钟位置，浏览器报Caution: request is not finished yet，导致seek失败。

初始化参数
var mediaDataSource = {
       type: 'mpegts',
       url: url,
       duration: 4558000,
       isLive: false,
};

let options = {
      hasVideo: true,
      hasAudio: ture,
      // enableWorker: true,
      isLive: false,
      withCredentials: false,
      // accurateSeek: true,
      seekType: 'range', // param
      cors: true,
      stashInitialSize: 1024 * 1024 * 3
      // rangeLoadZeroStart: true
}; // options

盼回复，谢谢

**Comment by xqq** [comment]
Seeking for mpegts is not supported, yet

**Comment by liminglu** [comment]
谢谢回复。
请问有计划支持么？

**Comment by xqq** [comment]
Yes I'm willing to support it but I don't have a schedule for now.
